development.ini

```diff
@@ -3,7 +3,22 @@
 # http://docs.pylonsproject.org/projects/pyramid/en/1.5-branch/narr/environment.html
 ###
 
-[app:main]
+[composite:main]
+use = egg:Paste#urlmap
+/ = consultapp
```

Comment:
Can we change `consult` to `public`?
Also I'd remove the `app` suffix in `xxxxxapp`.